cmd/analyzer/analyzer.go

```diff
@@ -149,6 +151,7 @@ func wipeStorage(cfg *config.StorageConfig) error {
 type Service struct {
 	analyzers         []SyncedAnalyzer
 	fastSyncAnalyzers []SyncedAnalyzer
+	helpers           []*http.Server
```

Comment:
Should this be just named (caching) proxies, at least while caching proxies are the only type of helpers there is?